content/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile.md

```diff
@@ -27,6 +27,8 @@ If you are part of an organization that uses SAML single sign-on (SSO), you won
 
 Issues, pull requests, and discussions will appear on your contribution graph if they were opened in a standalone repository, not a fork.
 
+Note that we limit the number of these items when displaying the contribution graph. If you've reached the limit, not all of your contribution might be accounted for.
```

Comment:
Small wording suggestion for clarity. And I think we can use the `{% data variables.product.company_short %}` variable to display `GitHub`. 

Let me know what you think of this change, @stefannibrasil, and I can help merge the content! 

```suggestion
{% data variables.product.company_short %} limits the number of these items when displaying the contribution graph. If you've reached the limit, the contribution graph may not display all of your contributions.
```

Reply:
oh, this reads much better. thank you @jc-clark 🎉